kalite/distributed/views.py

```diff
@@ -249,3 +250,15 @@ def handler_500(request):
         "value": unicode(value),
     }
     return HttpResponseServerError(render_to_string("distributed/500.html", context, context_instance=RequestContext(request)))
+
+def sphinx_docs(request, doc_url):
```

Comment:
I have not tried serving them statically. What I hoped to achieve was to serve the docs from an arbitrary url. But statically might be simpler.